src/transforms/corrcholeskylower.jl

```diff
@@ -0,0 +1,65 @@
+
+####
+#### correlation cholesky Lower
+####
+# Adapted from https://github.com/tpapp/TransformVariables.jl/blob/master/src/special_arrays.jl
+
+const TV = TransformVariables
+using LinearAlgebra
+
+"""
+    CorrCholeskyLower(n)
+Cholesky Lower of a correlation matrix of size `n`.
```

Comment:
This phrasing is odd.
```suggestion

Cholesky Lower of a correlation matrix of size `n`.
```

Reply:
Haha, that's my search and replace showing. It had been "Cholesky Factor" :)